src/wallet/rpcdump.cpp

```diff
@@ -72,6 +73,40 @@ std::string DecodeDumpString(const std::string &str) {
     return ret.str();
 }
 
+bool ImportPrivateKey(const string& strPrivkey, const string& strLabel)
```

Comment:
I would move up all auxiliary functions (the ones in title case), then the RPC handlers. WDYT?